<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>903</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>903</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Gestion du personnel</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Liste du personnel :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetPersonnel">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonGestionPersonnelAjouter">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/images/docs/add.png</normaloff>:/images/docs/add.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonGestionPersonnelModifier">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/images/docs/edit-undo.png</normaloff>:/images/docs/edit-undo.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonGestionPersonnelSupprimer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressources.qrc">
              <normaloff>:/images/docs/edit-delete.png</normaloff>:/images/docs/edit-delete.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gestion catalogue</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>845</width>
          <height>396</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QTableWidget" name="tableWidgetRayons">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>Rayons</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tableWidgetTypeProduits">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>Type Produits</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetProduits">
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>produits</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>prix</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>kilos</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>date d√©but</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>date fin</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="7" column="3">
            <widget class="QPushButton" name="pushButtonSupprimerProduits">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/images/docs/edit-delete.png</normaloff>:/images/docs/edit-delete.png</iconset>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="3">
            <widget class="QDateTimeEdit" name="dateTimeEditDebut">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Rayons :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBoxRayons"/>
           </item>
           <item row="0" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButtonAjoutRayons">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="ressources.qrc">
                 <normaloff>:/images/docs/add.png</normaloff>:/images/docs/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSupprimerRayons">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="ressources.qrc">
                 <normaloff>:/images/docs/list-remove.png</normaloff>:/images/docs/list-remove.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Type produits :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBoxTypeProduits"/>
           </item>
           <item row="1" column="3">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButtonAjoutTypeProduits">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="ressources.qrc">
                 <normaloff>:/images/docs/add.png</normaloff>:/images/docs/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonSupprimerTypeProduits">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="ressources.qrc">
                 <normaloff>:/images/docs/list-remove.png</normaloff>:/images/docs/list-remove.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonModifierTypeProduits">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="ressources.qrc">
                 <normaloff>:/images/docs/edit-undo.png</normaloff>:/images/docs/edit-undo.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Libelle produit :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEditLibelleProduit">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Prix : (au kilo)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Poids : </string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Date Fabrication:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Date Peremption :</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="3">
            <widget class="QDateTimeEdit" name="dateTimeEditFin">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="pushButtonModifierProduits">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="ressources.qrc">
               <normaloff>:/images/docs/edit-undo.png</normaloff>:/images/docs/edit-undo.png</iconset>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBoxPrix">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="singleStep">
              <double>0.250000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBoxPoids">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="singleStep">
              <double>0.250000000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;kilo(s)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;‚Ç¨/kilo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Producteur et Point de Vente</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="3">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Producteurs :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QTableWidget" name="tableWidgetProducteurs">
              <column>
               <property name="text">
                <string>Nom</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Prenom</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Mail</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Tel</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Adresse</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>ACCEPTER</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>REFUSER</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Points de Ventes :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QTableWidget" name="tableWidgetPointDeVentes">
              <column>
               <property name="text">
                <string>Nom</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Prenom</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Adresse</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>ACCEPTER</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>REFUSER</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Nom :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Etat :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Adresse :</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="labelNom">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelEtat">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelAdresse">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>GESTION VISITE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Planifications visites</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Gestion paie</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
